sdk/Cargo.toml

```diff
@@ -115,7 +115,7 @@ serde_bytes = "0.11.5"
 serde_cbor = "0.11.1"
 serde_derive = "1.0.197"
 serde_json = { version = "1.0.117", features = ["preserve_order"] }
-serde_with = "3.11.0"
+serde_with = { version = "3.12.0" }
```

Comment:
Why add the explicit `version` reference here? Generally, we don't go to the TOML map syntax unless we have features or some other requirement.

Reply:
I explored things that needed flags, will remove.